Tasks/filter.js

```diff
@@ -1,15 +1,13 @@
 // Filter array by type name
 
-Filter = (T, t) => {
-  remove = [];
-  for (C of T) {
-    x = T.indexOf(C);
-    if (typeof T[x] !== t) {
-      remove.unshift(x);
-    }
-  }
-  for (x of remove) T.splice(x, 1);
-  return T;
+// Step 5
+// Use Array.prototype.filter()
+
+'use strict';
+
+const filter = (array, type) => {
+  const filtered = array.filter((element) => typeof element === type);
+  return filtered;
 };
```

Comment:
```suggestion
const filter = (array, type) => array
  .filter((element) => typeof element === type);
```

Reply:
Reworked that 
https://github.com/M-Rybalko/Exams/tree/exam